index.html

```diff
@@ -0,0 +1,189 @@
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder">
+        <div class="text">
+          <div class="topic">Multi-Post Stories Gain+Glory</div>
+          <div class="row">
+            <div class="but"><span class="tab">Ruby on rails</span></div>
+            <div class="but"><span class="tab">css</span></div>
+            <div class="but"><span class="tab">JavaScript</span></div>
+            <div class="but"><span class="tab">html</span></div>
+          </div>
```

Comment:
- [ ] These are the list of project technologies So for this, you can use `ul` tag and `li` tag per each list. Please implement for better semantic HTML.

Reply:
completed